ui/src/components/mailboxes/UserMailboxes.vue

```diff
@@ -384,10 +384,14 @@ export default {
       // sort by quota
 
       return function (a, b) {
-        if (a.quota.value < b.quota.value) {
+        //we might have some undefined quota property
+        let varA = a.quota || { quota: {} };
+        let varB = b.quota || { quota: {} };
```

Comment:
```suggestion
        let varB = b.quota || {};
```
Same as previous comment